spec/unit/shoulda/matchers/action_controller/permit_matcher_spec.rb

```diff
@@ -180,6 +180,16 @@ def params_with_conditional_require(params, *filters)
       for(:show, verb: :get, params: { slug: 'foo' })
   end
 
+  it 'works when #fetch is used instead of #require (issue #495)' do
+    define_controller_with_strong_parameters(action: :create) do
+      params.fetch(:order, {}).permit(:eta, :diner_id)
+    end
+
+    expect(controller).
+      to permit(:eta, :diner_id).
+      for(:create, params: { order: { foo: '' } })
```

Comment:
Thanks for this -- this is definitely a more realistic way to test this. You're using `foo` as a key just so the `order` hash is non-empty, is that correct?